astrosat_users/models/models_customers.py

```diff
@@ -60,8 +78,16 @@ class Meta:
         max_length=64, choices=CustomerType.choices, default=CustomerType.MULTIPLE
     )
 
-    name = models.SlugField(unique=True, blank=False, null=False)
-    title = models.CharField(max_length=128, blank=False, null=False)
+    name = models.CharField( unique=True, blank=False, null=False, max_length=128)
+    official_name = models.CharField(unique=True, blank=True, null=True, max_length=128)
+    company_type = models.CharField(
+        choices=CompanyTypes.choices,
+        unique=True,
+        blank=True,
+        null=True,
+        max_length=64
+    )
+    registered_id = models.CharField(unique=True, blank=True, null=True, max_length=128)
```

Comment:
I _think_ it makes sense for this to be unique.  No two companies should have the same registered_id, right?

Reply:
That is my understanding, yes.